extractors/bilibili/bilibili.go

```diff
@@ -275,6 +275,12 @@ func (e *extractor) Extract(url string, option types.Options) ([]*types.Data, er
 	if err != nil {
 		return nil, err
 	}
+
+	// set thread number to 1 constantly to avoid http 412 error 
+	option.ThreadNumber = 1
+	fmt.Printf("Warning: Multi thread download is no longer supported by BiliBili, use single thread instead.\n")
+	fmt.Printf("Thread Number: %d\n", option.ThreadNumber)
```

Comment:
Basically LGTM, but I think this is unnecessary

Reply:
I agree that line 282 is unnessary. But I still insist that it may be confusing for users to see -n parameter not working as expected, so printing a warning is beneficial. 
But overall you are the owner, so you can make the final decision.

Reply:
OK then, I'll keep it.